indra/newview/llpanellandmarks.cpp

```diff
@@ -1047,6 +1047,85 @@ bool LLLandmarksPanel::handleDragAndDropToTrash(BOOL drop, EDragAndDropType carg
 	return true;
 }
 
+class LLLandmarkCopiedCallback : public LLInventoryCallback
+{
+public:
+    LLLandmarkCopiedCallback() {}
+
+    virtual void fire(const LLUUID& inv_item)
+    {
+        LLViewerInventoryItem* item = gInventory.getItem(inv_item);
+
+        if (item)
+        {
+            item->setComplete(TRUE);
+            item->updateServer(FALSE);
+
+            gInventory.updateItem(item);
+            gInventory.notifyObservers();
+        }
+
+        LLView::getWindow()->setCursor(UI_CURSOR_ARROW);
+    }
+};
```

Comment:
This looks incomplete, it's a landmark being copied into favorites' overflow, why isn't it calling setSortIndex to make sure position gets updated and saved?

Reply:
I also don't see any code for a position indicator when hovering over the overflow menu (like it was done for the main favorites panel)